brainiak/utils/utils.py

```diff
@@ -100,3 +103,164 @@ def fast_inv(a):
     except np.linalg.linalg.LinAlgError:
         logging.exception('Error from np.linalg.solve')
         raise
+
+
+def cov2corr(cov):
+    """Calculate the correlation matrix based on a
+        covariance matrix
+
+    Parameters
+    ----------
+
+    cov: 2D array
+
+    Returns
+    -------
+
+    corr: 2D array
+        correlation converted from the covarince matrix
+
+
+    """
+    assert cov.ndim == 2, 'covariance matrix should be 2D array'
+    inv_sd = 1 / np.sqrt(np.diag(cov))
+    corr = cov * inv_sd[None, :] * inv_sd[:, None]
+    return corr
+
+
+class read_design:
```

Comment:
Class names should be use the CapWords capitalization convention:
https://www.python.org/dev/peps/pep-0008/#class-names
While for the code that closely follows the formulas in the paper we can disregard this rule, please follow it here.